VueApp/src/styles/base.css

```
@@ -155,11 +246,22 @@ div.breadcrumbs {
 #mainLayoutHeaderSections a.q-btn.navLink {
 }
 
+/* Quasar's overlay-drawer glow bleeds a stray line down the left edge. */
+.q-drawer--mobile .q-layout__shadow:after {
+    box-shadow: none;
+}
+
 /*Left Nav styles*/
```

Stylelint (17.12.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)